packages/blockchain-link/src/workers/solana/index.ts

```diff
@@ -14,6 +14,34 @@ type Request<T> = T & Context;
 
 const TOKEN_PROGRAM_PUBLIC_KEY = new PublicKey('TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA');
 
+// This function matches the token mint to a name and symbol for some of the most popular tokens
+// Can be extended however desired, or replaced by a call to a token registry  API
+const getWhitelistedTokenInfo = (mint: string): { name: string; symbol: string } => {
```

Comment:
I would put this into a separate file, e.g. `src/workers/solana/tokenList`

Also we talked about having and option to fetch the token list... if not do it now, I would leave a PR comment proposing what can be done => "we can fetch token metadata from here and parse them you guys would prefer"